mlflow/deployments/base.py

```diff
@@ -271,7 +271,7 @@ def delete_endpoint(self, endpoint):
             "provided by specific target plugins."
         )
 
-    def list_endpoints(self):
+    def list_endpoints(self, page_token=None):
```

Comment:
tiny nit reminder to cleanup the docstring since pagination is going to be implemented.